### ISSUE TITLE
Designing an on-ramp for new mybinder.org operators

### ISSUE THREAD

**Issue by betatim** [jupyterhub/mybinder.org-deploy#1131]
A few times now the question has come up "how do I get involved with running mybinder.org" and right now there is no good answer. We should fix that and then create a page that explains it to people.

One of the hardest things is to take the first step, here some ideas from various people for what those first steps could be:
* start helping others on the binder gitter channel or forum
* start hanging out in the mybinder.org-deploy channel and chime in when there are problems
* start work on an open issue in this repo
* start work on an open issue in repo2docker or binderhub
* start regularly attending the team meeting

What else?

Some other resources on this topic:
* https://discourse.jupyter.org/t/the-operators-no-binder-isnt-forming-a-rock-band/694
* https://discourse.jupyter.org/t/interested-in-joining-the-mybinder-org-operations-team/761

**Comment by sgibson91** [comment]
Expressing interest in joining the Federation? I think the pathways there are either "donate a cluster and don't manage it" or "give access to the cluster that you manage". The second becomes easier if you are also a member of the operating team. Though with 1 (1.5?) data points, that's currently a tough trend to call.

**Comment by betatim** [comment]
Maybe another "first thing" people can do is create things like https://github.com/henchbot/. A small-ish project, using a set of skills you already have, self contained, solves a problem the team/community has, doesn't require long term commitment.

**Comment by betatim** [comment]
Is "joining the federation" a first step? I was thinking that would be something you might do as step two or three? Maybe something like the following as rough outline for one route of growing contributorship:
1. first steps
2. gain experience by helping run a existing cluster
3. get involved in multiple clusters and the federation
3. bring a new cluster and be responsible for it

**Comment by sgibson91** [comment]
Hmmm yeah, I'm thinking "motivations".

**Comment by choldgraf** [comment]
I agree w/ Sarah - I think "joining the federation" could be something like a shiny cool goal people can use to motivate themselves to get involved.

Other things people could do:

* Try deploying your own JupyterHub on Kubernetes and tell us how it went
* Write a Binder tutorial or blog post
* Make some improvements to the Binder documentation

also, I think the "motivations" idea could be fleshed out more too. A sub-section like, "why might I want to join the mybinder operations team?" could be helpful. Some reasons I can think of:

* Be part of a friendly, inclusive, international community
* Support an amazing and impactful experiment in open infrastructure
* Make connections with other Jovians across the world
* Improve your skills with Kubernetes and dev-ops in the cloud
* Learn how to solve problems with distributed teams with different time-zones
